src/colorizer/Dataset.ts

```diff
@@ -319,6 +319,7 @@ export default class Dataset {
     this.frameFiles = manifest.frames;
     this.outlierFile = manifest.outliers;
     this.metadata = { ...defaultMetadata, ...manifest.metadata };
+    console.log("Dataset metadata:", this.metadata);
```

Comment:
do you want to always log this?

Reply:
Yes, I'm planning on leaving it in for now.